library/std/src/collections/hash/map.rs

```diff
@@ -2948,9 +2948,8 @@ impl RandomState {
         // iteration order allows a form of DOS attack. To counter that we
         // increment one of the seeds on every RandomState creation, giving
         // every corresponding HashMap a different iteration order.
-        thread_local!(static KEYS: Cell<(u64, u64)> = {
-            Cell::new(sys::hashmap_random_keys())
-        });
+        thread_local!(static KEYS: Cell<(u64, u64)> = Cell::new(sys::hashmap_random_keys())
+        );
```

Comment:
that looks wrong

Reply:
Why do you think it's wrong?
[According to the macro doc](https://doc.rust-lang.org/std/macro.thread_local.html) It'd match this pattern
` ($(#[$attr:meta])* $vis:vis static $name:ident: $t:ty = $init:expr) => { ... }`

Reply:
it feels wrong to have the `);` in a separate line

either 
```rust
       thread_local!(static KEYS: Cell<(u64, u64)> = Cell::new(sys::hashmap_random_keys()));
```
or
```rust
        thread_local!(
            static KEYS: Cell<(u64, u64)> = Cell::new(sys::hashmap_random_keys())
        );
```

Reply:
right 👍